src/components/GamesContainer/GameCard/GameCardInfo/GameCardInfo.css

```diff
@@ -0,0 +1,61 @@
+.gameInfo {
+    position: relative;
+    top: -8px;
+    width: 91.18%;
+    padding-left: 4.41%;
+    padding-right: 4.41%;
+
+    .titleAndRanking {
```

Comment:
Here you have the same 4 lines of CSS as .gameDetails :

        display: flex;
        align-items: center;
        justify-content: space-between;
        width: 100%;

maybe you can combine them into one class.